src/objects.js

```diff
@@ -1,42 +1,56 @@
 const createPerson = (name, age) => {
-  // your code here
+  const person= {};
+  person.name=name;
+  person.age=age;
+  return person // your code here
 };
 
 const getName = (object) => {
-  // your code here
+  return object.name;// your code here
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];// your code here
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return object.hasOwnProperty(property)// your code here
 };
 
 const isOver65 = (person) => {
-  // your code here
+  if (person.age > 65){
+    return true}
+  else {return false} ; // your code here
 };
 
 const getAges = (people) => {
-  // your code here
+  return people.map(person => person.age);// your code here
 };
 
 const findByName = (name, people) => {
-  // your code here
+  const findPerson = people.filter(person => person.name === name);
+  return findPerson[0]// your code here
```

Comment:
Close. `find` is the method you are looking for: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find